multiqc/utils/search_patterns.yaml

```diff
@@ -232,6 +232,8 @@ methylQA:
 disambiguate:
     contents: 'unique species A pairs'
     num_lines: 2
+mirtop:
+    contents: 'mirtop'
```

Comment:
Is this just captured from standard out? This needs `shared: true` if so, as stdout log files are sometimes concatenated for some people.

Summary result files in JSON or YAML with consistent filenames are the best 😉

Reply:
@lpantano could best answer about how standard the file name will be / what would be the best patterns to match one. I did get the impression that less ambiguity would be better!

Reply:
Hi,

thabks for the comment. This tool will put the name mirtop on the top of the file, so it is not stdout.

said that, can you point me to an example of a tool with json in multiqc to try to adapt our tool output to that?

Thanks!